Doc/library/pdb.rst

```diff
@@ -639,6 +639,52 @@ can be overridden by the local file.
 
    Print the return value for the last return of the current function.
 
+.. pdbcommand:: exceptions [arg]
```

Comment:
To keep the consistency, we should probably write `exceptions [excnumber]` instead of `exception [arg]`. That's now `clear`/`disable` documents.